app/controllers/application_controller.rb

```diff
@@ -20,6 +20,10 @@ class ApplicationController < ActionController::Base
 
   before_action :set_timezone
 
+  # Allow users to impersonate other uses in a non-production environment.
+  # See the `pretender` gem.
+  impersonates :user
```

Comment:
You're right – it's probably well protected as it is. Thanks for clearing it up!